model/cat/pit_segments/operations.smithy

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+use opensearch.openapi#vendorExtensions
+
+@externalDocumentation(
+    "API Reference": "https://opensearch.org/docs/2.7/search-plugins/point-in-time-api/"
```

Comment:
Lets refer the documentation as https://opensearch.org/docs/latest/search-plugins/point-in-time-api/ rather than from version 2.7